File: Data Structures & Algorithms/sqrtx/submission-1.java
class Solution {
    public int mySqrt(int x) {
      if(x == 0) return 0;
      if(x == 1) return 1;

      int res = 1;
      for(int i = 1; i <= x / 2; i++) {
        
         if((long) i * i == x) {
            res = i;
            break;
         }

         if((long) i * i >= x) {
            res = i - 1;
            break;
         }   
      }

      return res;
    }
}

File: Data Structures & Algorithms/merge-k-sorted-linked-lists/submission-1.py
# Definition for singly-linked list.
# class ListNode:
#     def __init__(self, val=0, next=None):
#         self.val = val
#         self.next = next

class Solution:    
    def mergeKLists(self, lists: List[Optional[ListNode]]) -> Optional[ListNode]:
        res = []
        
        for i in range(len(lists)):
            while lists[i]:
                res.append(lists[i].val)
                lists[i] = lists[i].next

        res.sort()

        dummy = ListNode()
        curr = dummy

        for i in range(len(res)):
            curr.next = ListNode(res[i])
            curr = curr.next

        return dummy.next      
        
        